kbi/0008/index.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   triple: git-annex; uninit; remove dataset annex
+
+KBI0008: Remove a dataset's annex
+=================================
+
+:authors: Christian Mönch <christian.moench@web.de>
+:discussion: https://github.com/datalad-handbook/book/issues/939
+:keywords: annex, remove annex, annex uninit
```

Comment:
```suggestion
:keywords: annex, remove annex, annex uninit
:software-versions: datalad-0.18.3
```

Making concrete proposal for https://github.com/psychoinformatics-de/knowledge-base/pull/31#issuecomment-1517484884 Importantly, this tag should no be datalad-specific, because the knowledge base is not datalad-specific.

The topic of software version recording has come up before in other KBIs.